onnxruntime/core/providers/cuda/cuda_profiler.cc

```diff
@@ -93,7 +93,8 @@ void CUPTIAPI CudaProfiler::BufferCompleted(CUcontext, uint32_t, uint8_t* buffer
   free(buffer);
 }
 
-bool CudaProfiler::StartProfiling() {
+bool CudaProfiler::StartProfiling(TimePoint profiling_start_time) {
+  (void) profiling_start_time;
```

Comment:
You can use `ORT_UNUSED_PARAMETER` for this purpose.

Reply:
Thanks, done!